ifupdown2/addons/dhcp.py

```diff
@@ -61,6 +61,14 @@ def _up(self, ifaceobj):
             dhcp_wait = policymanager.policymanager_api.get_attr_default(
                 module_name=self.__class__.__name__, attr='dhcp-wait')
             wait = not str(dhcp_wait).lower() == "no"
+            inet6_ll_wait = policymanager.policymanager_api.get_iface_default(module_name=self.__class__.__name__, \
```

Comment:
To be consistent with the other policy attribute names, can we pls call it dhcp6-ll-wait